packages/webdriverio/tests/commands/browser/url.test.js

```diff
@@ -27,14 +27,39 @@ describe('url', () => {
             .toEqual({ url: 'http://foobar.com/foobar' })
     })
 
-    it('should throw an exception when a non-string value passed in', async () => {
+    it('should throw an exception when a non-string value passed in path', async () => {
```

Comment:
This test doesn't fail if no error was thrown, instead use expect(X).reject.toThrowError(Y)

Reply:
Same as above test

Reply:
I think what @erwinheitzman is saying is that if `browser.url(true)` doesn't fail the test passes even though it should fail. The solution here is to add

```js
expect.assertions(1)
```

to say we at least expect 1 assertion. This hasn't to do anything with your patch but we should fix it while we at it.